token.js

```diff
@@ -10,7 +10,7 @@ const generate = (identity) => {
 
     capability.addScope(new twilio.jwt.ClientCapability.IncomingClientScope(identity));
     capability.addScope(new twilio.jwt.ClientCapability.OutgoingClientScope({
-        applicationSid: config.twimlAppSid,
+        applicationSid: config.applicationSid,
```

Comment:
@flazz Do you mean a location other than this one?

Reply:
no, sorry, I was thinking there was a 3rd place, that 3rd place would be in the .env and .example.env which would mean changing the env var. that call is up to you.

Reply:
Documenting the gist of our verbal discussion: got it, I can conceive of us needing to change it later if we have multiple TwiML Apps so rather than try to make a new system for the env vars, we're going to YAGNI for right now.